server/app/uk/gov/ons/addressIndex/server/controllers/AddressController.scala

```diff
@@ -85,6 +89,16 @@ class AddressController @Inject()(val controllerComponents: ControllerComponents
     val epochVal = epoch.getOrElse("")
     val fromsourceVal = {if (fromsource.getOrElse("all").isEmpty) "all" else fromsource.getOrElse("all")}
 
+    val eboostVal = {if (eboost.getOrElse("1.0").isEmpty) "1.0" else eboost.getOrElse("1.0")}
```

Comment:
eboost is an `Option` that defaults to `None`. You could just do: `val eboostVal = eboost.getOrElse("1.0")`. See this test code: 
`object Main {
  def main(args: Array[String]): Unit = {
    val eboost = Some("1.2")
    // val eboost = None
    // val eboostVal = {if (eboost.getOrElse("1.0").isEmpty) "1.0" else eboost.getOrElse("1.0")}
    val eboostVal = eboost.getOrElse("1.0")
    println(eboostVal)
  }
}`